src/utils/uploadFileDivision.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const fs = require('fs');
+const crypto = require('crypto');
+
+const uploadFileDivision = (file) => {
+  if (!file) return null;
+
+  const uploadDir = './public/images/divisions/';
+
+  if (!fs.existsSync(uploadDir)) {
+    fs.mkdirSync(uploadDir, { recursive: true });
+  }
+
+  const ext = path.extname(file.name);
+
+  const generateRandomFilename = () => {
+    return crypto
+      .randomBytes(12)
+      .toString('base64')
+      .replace(/\+/g, 'A')
+      .replace(/\//g, 'B')
+      .replace(/=+$/, '')
+      .substring(0, 16);
+  };
```

Comment:
# Issue in this function
1. Acceptance Criteria Mismatch
The acceptance criteria says `The random string must consist of number, uppercase, and lowercase`

Current implementation:
- Doesn't guarantee presence of numbers, uppercase, and lowercase.

2. Length Calculation Issue
- Using 12 bytes in `randomBytes(12)` generates ~16 base64 characters
- After replacing characters and trimming, might produce less than 16 characters
- No guarantee of exact 16-character output